src/Microsoft.AspNet.Mvc.Razor.Host/MvcRazorHost.cs

```diff
@@ -273,5 +279,14 @@ public override ParserBase DecorateCodeParser([NotNull] ParserBase incomingCodeP
                     CreateModelExpressionMethodName = CreateModelExpressionMethod
                 });
         }
+
+        private IReadOnlyList<ChunkTree> GetInheritedChunkTrees(string sourceFileName)
+        {
+            var inheritedChunkTrees = GetInheritedChunkTreeResults(sourceFileName)
+                .Select(result => result.ChunkTree)
+                .ToList();
```

Comment:
`IReadOnlyList` indicates the collection is enumerable and has an indexer and `Count` property. Only need `IEnumerable<T>` to indicate ordering. (Detail of the ordering e.g. the lack of documented order for entries in a `Dictionary` is a separate question.)